src/crypto/utils/helpers.py

```diff
@@ -1,4 +1,13 @@
+import json
+
 from google.oauth2 import service_account
+from google.oauth2.service_account import Credentials
+
+
+def service_credentials(service_acct: dict) -> Credentials:
```

Comment:
I guess this is just a convenience function, right?

Reply:
yes, since used in a couple spots thought would move into it's on function and since names are long and a lot of chaining thought it might help improve readability

Reply:
makes sense, thanks